lightning/src/blinded_path/message.rs

```diff
@@ -13,6 +13,8 @@
 
 use bitcoin::secp256k1::{self, PublicKey, Secp256k1, SecretKey};
 
+use crate::ln::channelmanager::PaymentId;
+use crate::ln::msgs::DecodeError;
```

Comment:
Could you move these into the import block below, before `crate::ln::onion_utils`?

Reply:
Moved the imports to the appropriate location in [6ff2e49](https://github.com/lightningdevkit/rust-lightning/commit/6ff2e49123c3135dccea720d7104e6ad5b8f7308)
Thanks!